presto-docs/src/main/sphinx/release/release-0.226.rst

```diff
@@ -0,0 +1,20 @@
+=============
+Release 0.226
+=============
+
+General Changes
+_______________
+* Add peak running task count to statistics field of QueryCompletedEvent.
```

Comment:
put this at the end of general changes.  order of changes should be fixes, optimizations, then additions